src/node_postmortem_metadata.cc

```diff
@@ -76,61 +77,55 @@ int GenDebugSymbol();
 // namespace to avoid C++ name-mangling.
 
 #define NODE_CONST_POSTMORTEM_METADATA(V)                                     \
-  V(Environment, kContextEmbedderDataIndex, int)
+  V(Environment__kContextEmbedderDataIndex, int,                              \
+    Environment::kContextEmbedderDataIndex)                                   \
+  V(kNodeOffExtstrData, size_t, NODE_OFF_EXTSTR_DATA)
```

Comment:
Terrible name, `NODE_OFF_EXTSTR_DATA`... I admit I also don't really understand what purpose it serves since it's always the size of a pointer.